cdap-docs/reference-manual/source/http-restful-api/metadata.rst

```diff
@@ -15,6 +15,9 @@ Metadata HTTP RESTful API
 Use the CDAP Metadata HTTP RESTful API to set, retrieve, and delete the metadata annotations
 of applications, datasets, streams, and other entities in CDAP.
 
+**Note**: Metadata for versioned entities such as applications, programs, schedules and program runs is not versioned.
```

Comment:
Suggest:

**Note**: Metadata for versioned entities is not versioned, including entities such as applications, programs, schedules, and program runs. Additions to metadata in one version are reflected in all versions.

Reply:
Done.